inventory/server.lua

```diff
@@ -467,19 +467,7 @@ function RemoveInventory(inventoryId, item, amount, drop, player)
     else
         if PlayerData[inventoryId].inventory[item] - amount < 1 then
             UpdateUIInventory(player, inventoryId, item, 0)
-            UpdateUIInventory(inventoryId, inventoryId, item, 0)
```

Comment:
Pourquoi supprimer cette ligne ?

Reply:
@rdlh Cette ligne en particulier n'est pas du tout supprimé, mais je suppose que tu parles du codes après.
quand l'arme est équipé et qu'on l'a drop, ça fait fait bug le script et affiche une erreur.
En supprimant le bout de code, le script fonctionne normalement et aucun soucis notable n'apparait

Reply:
Est-ce si nous avons l'arme en mains elle est toujours enlevée quand on la supprime dans l'inventaire ? Ce code servais à ça il me semble.

Reply:
@GalaxHD551 Any update on this? ⬆️